### PR TITLE
Additions, rebalances and new roles

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

adds the valorian swordfighter a warrior style merc for valoria, adds the tiefling crimson marauder, a new skirmisher type fighter that may need rebalance since is totally new, adds updated descriptions and tutorials for the mercenary roles, reworks them to be 4 types of mercenary instead of 1-2-3 by adjusting their gear and skills on something more static instead of randomized skills, they are behind from a rare adventurer but they can keep themselves alive and have a fair gear, now covered in every possible area on a fairer way, being light armors on arms and neck and nerfed steel gear wich only garrison and rare adventurers should get, not dirty mercenaries who don't use the wide range of iron armors ingame and are able to upgrade to steel via gaming, except for our friend the zybantu mercenary who used to spawn with a longsword of 30 damage, full steel armor and heavy gloves for protection in every area, now gets most of his armor nerfed to fit alongside the other mercenaries who had their steel privilege removed except from steel helmets since they look cool and play mercenary can't be worse than a warrior adventurer 

the pr needs more testing and even more balance by the proper maintainers, im just a player and dev trowing my grain of " lets make this a bit more fair since is a mess!"

in other subject this pr also removes tieflings and drows from the garrison/maker's guild, voted by the players in exchange for these races being able to roll legendary smith/mason and be more grounded to the game lore of "tolerated species" instead of be inserted in roles where they shouldn't be normally, they would have to be hired ingame as workers or in the case of tieflings they are more of a "Savage band" of mercenaries that lift the suspiction of everyone (including the PTSD of Aasimars seeing again bronze devils after a millenia)

## Why It's Good For The Game

more content, more roles, more identity for the Tieflings and Drows

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

Captura de pantalla (2394)
new descriptions

Captura de pantalla (2406)

new Desert Rider Shield

Captura de pantalla (2407)

Backsprite

Captura de pantalla (2397)
Crimson Marauder

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm [hunk 1]
```diff
@@ -6,7 +6,14 @@
...
 
-	allowed_races = ALL_PLAYER_RACES_BY_NAME
+	allowed_races = list(
+		"Humen",
+		"Elf",
+		"Half-Elf",
+		"Dwarf",
+		"Aasimar"
+	)
+	allowed_sexes = list(MALE, FEMALE)
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm:16]: Don't need to add this.

File: code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm [hunk 2]
```diff
@@ -71,7 +78,14 @@
...
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm:88]: Ditto.

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/zybantine
-	name = "Zybantine"
-	tutorial = "A cutthroat from the western countries, you've headed into foreign lands to make even greater coin than you had prior."
+	name = "Zybantu Desert Rider"
+	tutorial = "A cutthroat from the western countries, you've Mastered the Art of Swordfighting alongside Polearms and Shields to storm enemy positions."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:3]: Don't capitalise Mastered, Art, Swordfighting, Polearms, Shields.

File: code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/steppesman
-	name = "Steppesman"
-	tutorial = "A mercenary from hailing from the wild frontier steppes. There are three things you value most; saigas, freedom, and coin."
+	name = "Steppesman Archer"
+	tutorial = "A mercenary from hailing from the wild frontier steppes. There are three things you value most; saigas, freedom, and coin They Use a Bow for quick attacks alongside a Saber for Close encounters"
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm:3]: Don't capitilise Bow, Saber, Close.

Also change saber to sabre.

File: code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/grenzelhoft
-	name = "Grenzelhoft"
-	tutorial = "A mercenary from the Grenzelhoft Empire's Mercenary Guild. Their only care is coin, and the procurement of coin."
+	name = "Grenzelhoftter Greatswordsman"
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm:2]: Two t's in Grenzelhofter.

File: code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
+	tutorial = "A mercenary from the Grenzelhoft Empire's Mercenary Guild. Their only care is coin, and the procurement of coin, they use a Heavy Iron Zweihander as Main Weapon, aim well every devastating hit with this heavy weapon."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm:3]: Don't capitalise Heavy, Iron, Zweihander, Main, Weapon.

**Review by BadAtThisGame302 — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm (hunk 5 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm:2]: is german and added it just due to that since it sounds cool

**Comment by sanshoom** [comment]
Is there anyway to change the Zybantine mercs and other related Zybantine classes to have a brighter shade of red for their clothing and keffiyeh?

**Comment by anything86** [comment]
Limiting a dodge class to +1 Speed when it's exclusive to a race that only gets +1 Speed racial is a headscratcher. Garrison fencer has potentially 2 speed more than Crimson Marauder, with a better, synergistic weapon. Valorian Swordsman is perhaps a worse offender since, even putting dodge aside, it has nothing to set itself apart or above the Crimson Marauder. Should maybe look into these things.

File: code/game/objects/items/rogueweapons/shields.dm [hunk 7]
```diff
@@ -157,6 +157,14 @@
 	icon_state = "spidershield"
 	coverage = 55
 
+/obj/item/rogueweapon/shield/tower/rider
+	name = "Desert Rider shield"
```
Comment (lavillastrangiato) [inline code/game/objects/items/rogueweapons/shields.dm:161]: probably change it to "desert rider shield", all-lowercase, so it remains in line with the general lowercase item names

File: code/game/objects/items/rogueweapons/shields.dm [hunk 8]
```diff
@@ -157,6 +157,14 @@
...
+	desc = "A iron reinforced shield that covers the entire chest, a design of the ziggurat popular on the desert mercenaries."
```
Comment (lavillastrangiato) [inline code/game/objects/items/rogueweapons/shields.dm:162]: try "A shield of Zybantine design, reinforced with iron and made to protect the upper body."

File: code/modules/clothing/rogueclothes/armor.dm [hunk 9]
```diff
@@ -75,6 +75,17 @@
 	body_parts_covered = COVERAGE_FULL
 	prevent_crits = ALL_EXCEPT_CHOP_AND_STAB
 
+//................ Crimson Marauder ............... //
+/obj/item/clothing/suit/roguetown/armor/leather/jacket/sea/marauder
+	slot_flags = ITEM_SLOT_ARMOR
+	name = "Jinete's Linothorax"
```
Comment (lavillastrangiato) [inline code/modules/clothing/rogueclothes/armor.dm:81]: "jinete's linothorax"

File: code/modules/clothing/rogueclothes/armor.dm [hunk 10]
```diff
@@ -75,6 +75,17 @@
...
+	desc = "An equipment styled in the shape of the skirmisher tiefling nomads of the Crimsonlands, a modified version of the ancestral craft of the original tiefling equipment of the Apotheosis war. Most of the armor has been stripped in favor of limberness and ease of replacement. The stained leather bears a red devil face making a war cry, a horrid ancestor of some blood-thirsty tiefling, no doubt."
```
Comment (lavillastrangiato) [inline code/modules/clothing/rogueclothes/armor.dm:82]: try "A stained leather cuirass bearing the snarling face of a demon. The tiefling nomads of the Crimsonlands shape this armor to hearken to that which their ancestors wore."

File: code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm [hunk 11]
```diff
@@ -1,40 +1,41 @@
 /datum/advclass/mercenary/grenzelhoft
-	name = "Grenzelhoft"
-	tutorial = "A mercenary from the Grenzelhoft Empire's Mercenary Guild. Their only care is coin, and the procurement of coin."
+	name = "Grenzelhoftter Greatswordsman"
+	tutorial = "A mercenary from the Grenzelhoft Empire's Mercenary Guild. Their only care is coin, and the procurement of coin, they use a heavy iron zweihander as main weapon, aim well every devastating hit with this heavy weapon."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/grenzelhoft.dm:3]: as a* main weapon and aim well...

File: code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm [hunk 12]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/steppesman
-	name = "Steppesman"
-	tutorial = "A mercenary from hailing from the wild frontier steppes. There are three things you value most; saigas, freedom, and coin."
+	name = "Steppesman Archer"
+	tutorial = "A mercenary from hailing from the wild frontier steppes. There are three things you value most; saigas, freedom, and coin They Use a bow for quick attacks alongside a sabre for close encounters"
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm:3]: Don't capitalize They and Use.

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm [hunk 13]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/zybantine
-	name = "Zybantine"
-	tutorial = "A cutthroat from the western countries, you've headed into foreign lands to make even greater coin than you had prior."
+	name = "Zybantu Desert Rider"
+	tutorial = "A cutthroat from the western countries, you've Mastered the art of swordfighting alongside polearms and shields to storm enemy positions."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:3]: Don't capitalize Mastered.

File: code/modules/clothing/rogueclothes/armor.dm [hunk 14]
```diff
@@ -75,6 +75,17 @@
 	body_parts_covered = COVERAGE_FULL
 	prevent_crits = ALL_EXCEPT_CHOP_AND_STAB
 
+//................ Crimson Marauder ............... //
+/obj/item/clothing/suit/roguetown/armor/leather/jacket/sea/marauder
+	slot_flags = ITEM_SLOT_ARMOR
```
Comment (NPC1314) [inline code/modules/clothing/rogueclothes/armor.dm:80]: check the parent var before adding please

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm [hunk 15]
```diff
@@ -13,45 +13,44 @@
...
+		H.mind.adjust_skillrank(/datum/skill/combat/shields, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)//so they can heal themselves, after all they didn't survived the mercenary life without know how to sew their wou
+		H.mind.adjust_skillrank(/datum/skill/misc/riding, 3, TRUE)//the desert riders didn't knew how to ride... until now
 
-		H.merctype = 1
 
 		H.change_stat("strength", 1)
+		H.change_stat("constitution", 1)//so they get 3 stat points like anyone else
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:54]: 4, with the weight of STR this if 5 points bonus too, though other merc classes more extreme

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm [hunk 16]
```diff
@@ -13,45 +13,44 @@
...
-		H.mind.adjust_skillrank(/datum/skill/combat/shields, pick(0,1,1), TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/shields, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)//so they can heal themselves, after all they didn't survived the mercenary life without know how to sew their wou
+		H.mind.adjust_skillrank(/datum/skill/misc/riding, 3, TRUE)//the desert riders didn't knew how to ride... until now
 
-		H.merctype = 1
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:53]: why is this deleted and what does it do?

**Comment by ThePainkiller** [comment]
To be honest these sound like such long titles, sometimes less is more

File: code/modules/jobs/job_types/roguetown/other/merc_classes/valorian.dm [hunk 17]
```diff
@@ -0,0 +1,45 @@
+/datum/advclass/mercenary/valoria
+	name = "Valorian Swordfighter"
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/valorian.dm:2]: change do just "Duelist"

File: code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm [hunk 18]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/steppesman
-	name = "Steppesman"
-	tutorial = "A mercenary from hailing from the wild frontier steppes. There are three things you value most; saigas, freedom, and coin."
+	name = "Steppesman Archer"
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm:2]: back to just "Steppesman"

File: code/modules/jobs/job_types/roguetown/other/merc_classes/boltslinger.dm [hunk 19]
```diff
@@ -1,6 +1,6 @@
 /datum/advclass/mercenary/boltslinger
-	name = "Boltslinger"
-	tutorial = "A cutthroat and a soldier of fortune, your mastery of the crossbow has brought you to many battlefields, all in pursuit of mammon."
+	name = "Boltslinger Sharpshooter"
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/other/merc_classes/boltslinger.dm:2]: Back to "Boltslinger"

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm (hunk 16 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:53]: it limits mercenary slots, removed from every mercenary so they can be repeated even if an admin gives more slots for the role since the mercenary role is 4 guys

File: code/modules/jobs/job_types/roguetown/other/merc_classes/valorian.dm (hunk 17 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/other/merc_classes/valorian.dm:2]: he isn't a duelist but a valorian warrior in lore and code, duelist classes aren't planned for the valorian humen but for the elf, wich i ain't coding soon

File: code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm (hunk 18 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/other/merc_classes/steppesman.dm:2]: is a description for new people to know what does every role and giving them some flavour

**Comment by Dorodomki** [comment]
> Grenzelhofter Greatswordsman should just be Landsknecht anyways, leaning so hard on the theme the name works better and less clumsy.
> 
> Crimson Marauder is not perfect but it works, maybe theres a better name for them though? Take it up on discord.
> 
> Moved from TM since the sword conflicts with the general weapon fix PR that got higher priority. Got several other things to attend to in this PR before it can be merged in any case.


The Grenzelhoffter Greatswordsman is the new name given by people in discord since it sounds more german, landsknecht is too normal? not my taste for it

File: code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm (hunk 15 above)
Comment (NPC1314) [reply code/modules/jobs/job_types/roguetown/other/merc_classes/zybantine.dm:54]: Comment is wrong, fix it.

**Comment by NPC1314** [comment]
Names still coal, needs fix.

**Comment by Dorodomki** [comment]
> Names still coal, needs fix.

@NPC1314 

wich ones? the grenzelhoft was chosen by pop and the other explain the class a bit

**Comment by NPC1314** [comment]
I guess the only truly abominable one is the boltslinger sharpshooter one, though all of them are too wordy and long IMO

**Review by NPC1314 — APPROVED** [review]
